### PR TITLE
Romanian Translation

### PR DESCRIPTION
**Pull request by b-dorian**
Added Romanian translation also to the new version of Private Kit.

### CODE REVIEW

File: app/locales/ro.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    "push_at_risk_message": "V-ați intersectat cu un pacient COVID-19",
+    "push_at_risk_title": "Factor de risc identificat",
+    "see_exposure_history": "Vezi istoricul expunerilor",
+    "settings_title": "Tablou de bord",
+    "share_location_data": "Trimite locațiile",
+    "show_overlap": "Accesează datele publice",
+    "team": "Echipa",
+    "team_para": "Echipa noastră este formata dintr-un consorțiu de epidemiologi, ingineri, oameni de stiință de date, promotori a sigurantei datelor personale, cat si profesori și cercetatori de la instituții cu reputație, ce includ: MIT, Harvard, Clinica Mayo, TripleBlind, EyeNetra, Ernst & Young și Link Ventures.",
+    "terms_of_use": "Termeni de folosire",
+    "terms_of_use_url": "https://docs.google.com/document/d/1mtdal_pywsKZVMXLHjjj5eKznipPLP8sM1HwFTIhjo0/edit#",
```
Comment (alpita-masurkar) [inline app/locales/ro.json:98]: remove all the 3 URL keys. they will fall back to the default.

File: app/locales/ro.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    "event_history_title": "Istoricul expunerilor",
+    "export_para_1": "Dacă sunteți diagnosticat pozitiv cu COVID-19, va rugam sa împărțiți istoricul traseelor pentru a ajuta la identificarea persoanelor ce au fost posibil infectate de dumneavoastra,",
+    "export_para_2": "Traseele sunt exportate ca o lista simpla ce contine latitudine, longitudine și timp, fără nici o alta informație de natura personala.",
+    "home_at_risk_header": "Expunere Posibilă",
+    "home_at_risk_subsubtext": "Asta nu înseamna ca sunteți sigur infectat,",
+    "home_at_risk_subtext": "Având în vedere istoricul dumneavoastra GPS, este posibil sa va fi intersectat cu cineva găsit pozitiv cu COVID-19.",
+    "home_enable_location": "Activați Datele Despre Locație",
+    "home_mayo_link_heading": "Mai multe informatii despre COVID-19",
+    "home_mayo_link_label": "de la Ministerul Sănătății",
+    "home_mayo_link_URL": "http://www.ms.ro/coronavirus-covid-19/",
```
Comment (alpita-masurkar) [inline app/locales/ro.json:30]: @b-dorian @tstirrat should this link take some customized local value?

File: app/locales/ro.json [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    "authorities_removal_alert_desc": "Sunteți sigur ca vreți sa ștergeți aceasta sursa?",
+    "authorities_removal_alert_proceed": "Șterge",
+    "authorities_removal_alert_title": "Șterge Sursa",
+    "authorities_title": "Surse de Încredere",
+    "choose_provider_subtitle": "Pentru informatii despre expuneri, trebuie sa va abonati la o autoritate de sănătate.",
+    "choose_provider_title": "Alege autoritate sănătate",
+    "commitment": "Angajament",
+    "commitment_para": "Safe Paths verifica în deplina siguranta intersectările cu alte persoane folosind locația ta. Datele tale nu vor parași NICIODATA telefonul fără aprobarea ta.",
+    "default_news_site_name": "Știri Safe Paths",
+    "default_news_site_url": "https://covidsafepaths.org/in-app-news",
```
Comment (alpita-masurkar) [inline app/locales/ro.json:19]: remove URL

File: app/locales/ro.json (hunk 2 above)
Comment (tstirrat) [reply app/locales/ro.json:30]: @penrods this is overwritten with non mayo link on the main screen. Is that what we want to offer as an option here?

Comment (tstirrat) [reply app/locales/ro.json:29]: I would say _for now_, keep this as the Mayo Clinic link, until we decide on whether/how to include other links.

Comment (b-dorian) [reply app/locales/ro.json]: Changed translation back to Mayo Clinic, removed URL key so it defaults

File: app/locales/ro.json (hunk 1 above)
Comment (b-dorian) [reply app/locales/ro.json]: removed

File: app/locales/ro.json (hunk 3 above)
Comment (b-dorian) [reply app/locales/ro.json]: removed

**Comment by b-dorian** [comment]
removed mayo URL key so it defaults to actual mayo link as requested in the change

**Comment by tstirrat** [comment]
The diff is not clean anymore, you'll need to resync with `develop`

**Comment by b-dorian** [comment]
synced

**Comment by tstirrat** [comment]
There are still 14 files changed.

try

```
git remote add upstream git@github.com:tripleblindmarket/covid-safe-paths.git
git fetch upstream
git merge upstream/develop
```

**Pull request merged**